downstream/modules/automation-hub/proc-obtaining-org-collection-url.adoc

```diff
@@ -3,18 +3,18 @@
 [id="proc-create-api-token"]
 = Retrieving your Red Hat Certified Collections Sync URL and API token.
 
-You can sync Red Hat certified collections curated by your organization from console.redhat.com to your local {HubName}.
+You can sync Red Hat Ansible Certified Content Collections curated by your organization from `{Console}` to your {PrivateHubName}.
 
 .Prerequisites
 
-* You have Organization Administrator permissions for console.redhat.com.
+* You have Organization Administrator permissions for {Console}.
```

Comment:
You don't need an org admin account to sync. The org admin account is only required to create the synclist on console.redhat.com. Once the synclist is updated any account can be used to configure automation hub.